### ISSUE TITLE
Verify user's data cleaning after logout

### ISSUE THREAD

**Issue by lorenzo-cavazzi** [SwissDataScienceCenter/renku-ui#452]
It seems that some user data are not cleaned correctly during the logout. This allows another user logging in later to the same browser to access previous user's data.
This seems to affect only additional namespaces in renku -- e.g. it's happening in `lorenzo.dev.renku.ch` but not in `dev.renku.ch`. 

To replicate, log in to any `dev.renku.ch` subdomain, start at least one notebook server, then logout and login with a different user. It should be possible to see the previous user's servers in the `Notebooks` tab

**Comment by lorenzo-cavazzi** [comment]
I can't reproduce the problem with notebooks anymore.
Anyway, it seems that the `session` cookie is not removed during the logout. As a consequence, when logging in from the same browser with a different user, the `session` cookie value doesn't change

**Comment by ciyer** [comment]
@lorenzo-cavazzi Can you check if this is still occurring? If not, I think we can close the issue.

**Comment by lorenzo-cavazzi** [comment]
I can confirm this was recently solved by this PR SwissDataScienceCenter/renku-gateway#320 , and the problem was basically this one SwissDataScienceCenter/renku-gateway#314 .

TLDR: the user was not logged out properly from JupyterHub, leading to strange behaviors when trying to log in with another user in the same browser. The `session` cookie can live in the UI without issues since it's a way to identify the client, and the "real* tokens are stored in the gateway and cleaned properly
